### DISCUSSION TITLE
Copilot does not automatically display suggestions

### DISCUSSION THREAD

**Discussion by ElmaK141** [community/community#41023] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When I try and prompt Copilot for a suggestion, instead of displaying the suggestion as soon as I move to the next line as I had seen in numerous videos on the topic, it doesn't show the suggestion automatically. Once I input something though, the suggestion pops up. Here's a couple examples:

When I write "#q: How to run this file?" and hit enter, I expect the suggestion to show up immediately, instead is displays like this
![image](https://user-images.githubusercontent.com/47115742/206241879-6d886674-983d-443e-98bb-34e882db5b5d.png)
Once I type in "#", the suggestion pops up
![image](https://user-images.githubusercontent.com/47115742/206242049-ae71b272-ef5b-45d1-94c3-2fb7b781a3e2.png)

Same with suggesting functions
![image](https://user-images.githubusercontent.com/47115742/206242181-483062ba-decf-4abd-9fc6-5fd3a048e9b2.png)
Nothing, but when in input "def"
![image](https://user-images.githubusercontent.com/47115742/206242322-bcdebe60-f15e-4c26-ba21-3037a6619354.png)

Oddly enough, it doesn't even have to be what the suggestion is looking for.
For example, with the run file example above, I can also just type anything, delete it, and now the suggestion appears.

It's almost as if the next line needs to be interacted with in some way for the suggestions to appear.

When I move to the new line, the output shows 
![image](https://user-images.githubusercontent.com/47115742/206244046-996ebb98-c9b1-4c5d-a477-b30ff9e882d7.png)
suggesting it works. When I input anything the same line on the bottom is output and now it works
![image](https://user-images.githubusercontent.com/47115742/206244195-a6a366a7-2dc9-4340-8517-089030daf48a.png)

I know it's not a version or firewall, checked all that already. I'm thinking there's some setting in VS Code that is causing this that I need to disable. Already tried Intellicode, intellisense, suggestions. About anything I can think of.

Any help would be appreciated

Update: It seems this lack of instant suggestion is only present when I'm editing python files

**Comment by ossanna16** [comment]
Hi there @ElmaK141 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by oshapio** [comment]
@ElmaK141 Did you solve it?

**Reply by ElmaK141** [reply]
Nope! Still persistent to this day
